examples/blog/post.js

```diff
@@ -0,0 +1,162 @@
+var Minio = require('minio')
+
+var bucketExistsOrCreate = function (client, bucket) {
+    return new Promise((fulfill, reject) => {
+        client.bucketExists(bucket, err => {
+            if (err) {
+                if (err.code != 'NoSuchBucket') {
+                    return reject(`error checking bucket existence: ${err}`)
+                }
+                client.makeBucket(bucket, '', err => {
+                    if (err) {
+                        return reject(`error making bucket: ${err}`)
+                    }
+                    fulfill()
+                })
+            }
+            fulfill()
+        })
+    })
+}
+
+module.exports = function (context, params) {
+
+    let op = params["op"]
+    let post = params["post"]
+
+    // BEGIN of workaround:
+
+    // importing secret from api-gateway is not supported
```

Comment:
I commented on the bug, but couldn't we inject this secret when we define the function?